rsocket-examples/src/test/java/io/rsocket/integration/IntegrationTest.java

```diff
@@ -90,6 +93,8 @@ public void startup() {
 
     server =
```

Comment:
What about creating two+ servers and two+ clients and being able to use different interceptors for them?